ibridges/irodsconnector/session.py

```diff
@@ -37,10 +37,30 @@ def __init__(self, irods_env: Optional[dict] = None, irods_env_path: Optional[st
             with open(env_fp, "r", encoding="utf-8") as f:
                 irods_env = json.load(f)
 
+
         self._password = password
         self._irods_env = irods_env
         self._irods_env_path = irods_env_path
         self._irods_session = self.connect()
+        if irods_home is not None:
+            self._irods_env["irods_home"] = irods_home
+        if "irods_home" not in self._irods_env:
+            self._irods_env["irods_home"] = '/'+self.zone+'/home/'+self.username
+
+    def home(self) -> str:
```

Comment:
Can we also make a home setter? When you are connected to iRODS and want to change your current working directory (irods_home) and you do not want to create a new session.

Reply:
Done!